iron-ajax.html

```diff
@@ -353,13 +353,15 @@
         contentType = 'application/x-www-form-urlencoded';
       }
       if (contentType) {
-        headers['Content-Type'] = contentType;
+        headers['content-type'] = contentType;
       }
       var header;
 
       if (this.headers instanceof Object) {
         for (header in this.headers) {
-          headers[header] = this.headers[header].toString();
+          // Normalize headers in lower case to make it easier for iron-request
+          // to reason about them.
+          headers[header.toLowerCase()] = this.headers[header].toString();
```

Comment:
@rictic probably not supposed to be relying on iron-request anyway. Do we have any way to mark an element as private? Could be useful so people know what they can/should rely on. Thank you!